packages/renderer-bootstrap4/renderers/table-linked-struct-field-renderer.tsx

```diff
@@ -87,6 +81,16 @@ const Bootstrap4TableLinkedStructFieldRenderer = ({
           )}
         </tbody>
       </table>
+      <div className={createCssClass(cssName, 'controls')}>
```

Comment:
Why was this moved to underneath the table? If this is an application specific change, we can talk through how to create a custom renderer.

Reply:
In general, I think it's a more intuitive place for the button was application-specific.